### PR TITLE
log

### PR DESCRIPTION
**Pull request by zeitlinger**

### CODE REVIEW

File: viewer/src/data/index.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ export interface ButtonData {
 
 export type MapData = GaiaHex[];
 export type HighlightHexData = Map<GaiaHex, { cost?: string }>;
-export type LogPlacement = "top" | "bottom" | "off";
+export type LogPlacement = "top" | "top-full" | "bottom" | "bottom-full" | "off";
```
Comment (coyotte508) [inline viewer/src/data/index.ts:35]: I would keep the `full/recent` setting inside the viewer itself. We may want to switch between the two to check the past info of the game, without needing to go to the sidebar.

The placement is a more permanent setting, so it makes sense to keep it in the sidebar and remove the control in the viewer

Comment (zeitlinger) [reply viewer/src/data/index.ts:35]: the problem was that the log keeps changing to "recent" when I replay a log - is there another way to avoid that?

Comment (zeitlinger) [reply viewer/src/data/index.ts:35]: changed it back now - and it's still working - so it was probably something else.

File: viewer/src/components/Game.vue [hunk 2]
```diff
@@ -61,7 +61,7 @@
         />
       </template>
       <Pool class="col-12 order-10 mt-4" />
-      <AdvancedLog class="col-12 order-last mt-4" :currentMove="currentMove" v-if="logPlacement !== 'top'" />
+      <AdvancedLog class="col-12 order-last mt-4" :currentMove="currentMove" v-if="!topLog" />
```
Comment (coyotte508) [inline viewer/src/components/Game.vue:64]: `logPlacement === 'bottom'`

Then we don't need any special treatment in `AdvancedLog`

File: viewer/src/components/AdvancedLog.vue [hunk 3]
```diff
@@ -1,25 +1,26 @@
 <template>
-  <div v-if="gameData.phase !== 'setupInit'">
+  <div v-if="gameData.phase !== 'setupInit' && placement !== 'off'">
```
Comment (coyotte508) [inline viewer/src/components/AdvancedLog.vue:2]: No need to add `placement` here if previous comment taken into account

**Review by coyotte508 — CHANGES_REQUESTED** [review]

**Pull request merged**